reconcile/gitlab_housekeeping.py

```diff
@@ -352,7 +352,8 @@ def get_merge_requests(
         item = {
             "mr": mr,
             "labels": labels,
-            "label_priority": label_priotiry,
+            "label_priority": label_priority,
+            "priority": f"{label_priority} - {MERGE_LABELS_PRIORITY[label_priority]}",
```

Comment:
so the label value would be something like `1 - approved: critical`, `2 - approved: high` etc? this is nice. in case the we add new priorities because, e.g. `1 - approved: ultra-high` because we could then see that critical would only be 2nd anymore, e.g. `2 - approved: critical`.

same could be achieved by having the absolute numberic priority and the humand readable priorty as a two dedicated labels. depends on what we want to do with these values in dashboards or in alerts.

just mumbling through my first coffee, ignore me if it does not make any sense.

Reply:
side note: i think the label would be `1 - critical`, no?

to the point, i chose a single label so it can be a single column in a table, but i'm open to improving in a future iteration.